### PR TITLE
Run with PowerShell button on toolbar

### LINKED ISSUE #7755: Add "Run script" button to toolbar when ps1 files are selected
### What's the Problem?

Double clicking a ps1 file will open the script in the default editor, to run the script, you need to right click and find the "Run with PowerShell" option in the context menu.

### Solution/Idea

Adding a "Run script" button to the toolbar will make running PowerShell scripts easier and improve the users workflow.

### Alternatives

NA

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will add a run script button to the toolbar when ps1 files are selected | Must |

### Files Version

_No response_

### Windows Version

_No response_

### Comments

_No response_

### PR DESCRIPTION
**Pull request by puppetsw**
**Resolved / Related Issues**
- Closes #7755 

**Details of Changes**
Add details of changes here.
- Added "Run with PowerShell" button on toolbar when selecting ps1 script file.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Example2

### CODE REVIEW

File: src/Files/UserControls/InnerNavigationToolbar.xaml [hunk 1]
```diff
@@ -213,6 +213,18 @@
...
                     </AppBarButton.Icon>
                 </AppBarButton>
+                <AppBarButton
+                    x:Name="RunWithPowerShellButton"
+                    Command="{x:Bind ViewModel.RunWithPowerShellCommand, Mode=OneWay}"
+                    IsEnabled="{x:Bind ViewModel.IsPowerShellScript, Mode=OneWay}"
+                    Label="{helpers:ResourceString Name=RunWithPowerShell}"
+                    LabelPosition="Default"
+                    ToolTipService.ToolTip="{helpers:ResourceString Name=RunWithPowerShell}"
+                    Visibility="{x:Bind ViewModel.IsPowerShellScript, Mode=OneWay, FallbackValue=Collapsed}">
```
Comment (yaichenbaum) [inline src/Files/UserControls/InnerNavigationToolbar.xaml:223]:
```suggestion
                    x:Load="{x:Bind ViewModel.IsPowerShellScript, Mode=OneWay, FallbackValue=Collapsed}">
```

File: src/Files/UserControls/InnerNavigationToolbar.xaml [hunk 2]
```diff
@@ -213,6 +213,18 @@
                         <FontIcon FontFamily="{StaticResource RecycleBinIcons}" Glyph="&#xEF88;" />
                     </AppBarButton.Icon>
                 </AppBarButton>
+                <AppBarButton
+                    x:Name="RunWithPowerShellButton"
+                    Command="{x:Bind ViewModel.RunWithPowerShellCommand, Mode=OneWay}"
+                    IsEnabled="{x:Bind ViewModel.IsPowerShellScript, Mode=OneWay}"
```
Comment (yaichenbaum) [inline src/Files/UserControls/InnerNavigationToolbar.xaml:219]: Is this needed if the button is already hidden?

File: src/Files/UserControls/InnerNavigationToolbar.xaml [hunk 3]
```diff
@@ -213,6 +213,18 @@
...
+                    Label="{helpers:ResourceString Name=RunWithPowerShell}"
```
Comment (yaichenbaum) [inline src/Files/UserControls/InnerNavigationToolbar.xaml:220]: Can you add this string to the en-us resource file (we'll generate the the strings in the other files later one)?

File: src/Files/UserControls/InnerNavigationToolbar.xaml (hunk 2 above)
Comment (puppetsw) [reply src/Files/UserControls/InnerNavigationToolbar.xaml:220]: No not needed, I can remove it.

**Review by yaichenbaum — DISMISSED** [review]
Thank you!

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**